src/lua/compat.c

```diff
@@ -85,6 +85,6 @@ static const struct luaL_Reg internal_compat[] = {
 int
```

Comment:
Quoted @Buristan's comment to discuss it in a thread:

> > The module can't be pulled as require('compat'), because it is
> > tarantool specific tuning mechanism, not a general compatibility module.
>
> Typo: s/tarantool specific tuning mechanism/the tarantool specific tuning mechanism/

Reply:
Added the indefinite artile. It fits better here AFAIU.